packages/app/ui/components/EditProfile/EditAttestationsDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+import * as db from '@tloncorp/shared/db';
+import { Pressable } from '@tloncorp/ui';
+
+import { Field } from '../Form';
+import { ListItem } from '../ListItem';
+import { WidgetPane } from '../WidgetPane';
+
+export function EditAttestationsDisplay(props: {
+  attestations: db.Verification[];
+  onPressAttestation?: (attestation: 'phone' | 'twitter') => void;
+}) {
+  console.log(`attestations editor`, props.attestations);
+  const twitterAttestation = props.attestations.find(
```

Comment:
I notice we do this in a few places. Are we fine with the first result being "the" one? can you have multiple?

Reply:
Yes, explicit decision. We don't plan to support verifying multiple phone numbers or twitter accounts from the client anytime soon.